Pidgin/Parser.Number.cs

```diff
@@ -143,5 +145,54 @@ private static long GetLetterOrDigitValueLong(char c)
         }
         private static long GetUpperLetterOffsetLong(char c) => c - 'A';
         private static long GetLowerLetterOffsetLong(char c) => c - 'a';
+		
+		/// <summary>
+        /// A parser which parses a floating point number with an optional sign.
+        /// The resulting <c>double</c> is not checked for overflow.
+        /// </summary>
+        /// <returns>A parser which parses a floating point number with an optional sign</returns>
+        public static Parser<char, double> Float { get; } = RealNum();
+
+        /// <summary>
+        /// A parser which parses a floating point number with an optional sign.
+        /// The resulting <c>double</c> is not checked for overflow.
+        /// </summary>
+        /// <returns>A parser which parses a floating point number with an optional sign</returns>
+        public static Parser<char, double> RealNum()
+            => Map(
+                (sign, num) => sign.HasValue && sign.Value == '-' ? -num : num,
+                (Char('-').Or(Char('+'))).Optional(),
+                UnsignedReal()
+            ).Labelled($"real number");
+			
+		/// <summary>
+		/// A parser which parses a floating point number without a sign.
+		/// The resulting <c>double</c> is not checked for overflow.
+		/// </summary>
+		/// <returns>A parser which parses a floating point number without a sign.</returns>
+
+		public static Parser<char, double> UnsignedReal()
```

Comment:
Nit pick: Indentation looks a little wonky

Reply:
Cut and paste error, will resolve in next upload.